timescaledb/tutorials/analyze-intraday-stocks/explore-stocks-data.md

```diff
@@ -0,0 +1,286 @@
+# 3. Explore stock market data
+
+Now that you've successfully collected 1-min intraday stock data, it's time to have some fun and explore the 
+data.
+
+Because of the high granularity of the dataset, there are numerous ways to explore it. For example, you could analyze stock prices and volumes on a minute-by-minute basis. With TimescaleDB, you could also bucket records into custom intervals (e.g. 2-min or 15-min) fairly easily and use TimescaleDB aggregating functions. 
```

Comment:
```suggestion
Because of the high granularity of the dataset, there are numerous ways to explore it. For example, you could analyze stock prices and volumes on a minute-by-minute basis. With TimescaleDB, you could also bucket records into custom intervals (e.g. 2-min or 15-min) using TimescaleDB aggregate functions. 
```